ruby-challenges.rb

```diff
@@ -31,3 +52,17 @@
 # Expected output: 'LEARN is not a palindrome'
 palindrome_tester3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+# Use casecmp to ignore case.
+
+def palindrome?(string)
+    if string.downcase == string.downcase.reverse
+      "#{string} is a palindrome"
+    else
+      "#{string} is not a palindrome"
+    end
+end
+
+p palindrome?(palindrome_tester1)
+p palindrome?(palindrome_tester2)
+p palindrome?(palindrome_tester3)
```

Comment:
Nice work!  Crushed this section!  If you want to stretch yourself, consider edge cases when applicable.